_partners/squareshift.md

```diff
@@ -0,0 +1,6 @@
+---
+name: SquareShift
+# upload your logo to the following directory - must be square
+logo: 'https://static.wixstatic.com/media/4d2638_9099aa345f0848a18f4d8ebf24ae0fb5~mv2.png'
```

Comment:
Please submit the logo directly as part of the pull request - example:

logo: '/assets/media/partners/squareshift.png'

Reply:
I initially mis-understood that relative url has to be on our domain, then testing locally help identify that it has to be relative to Opensearch site. This has been addressed now. Thank you

Reply:
Looks good!